### PR TITLE
Feat/update assets based on feedback

### LINKED ISSUE #562: View assets per key additional functionality
## Story

Fix now:

**As a** user
**I want** to see an error state
**So that** I am informed why the view is not working

As a user
I want to see a reasonable empty state
So that I am informed about how to deposit assets to vega and which markets I can trade in

**As a** user
**I want** the market lozenges in the expanded view
**So that** the view is not so busy

Split to other tickets

#576 
#577 

## Acceptance Criteria

Split out into other tickets:

Unit test:
- [ ] Some version of this expanded view is shown when hovering over the asset too
- [ ] Once expanded there are buttons / icons prompting me to deposit / withdraw / transfer that asset which take me to the relevant screen in console - probably initially the side bar but eventually the guided flow, and ideally with the asset already set in each case assuming Console can support this in time. Relies on #502 which relies on console deploying to mainnet.
- [ ] If I have no funds, by default USDT and VEGA are displayed with the same expansion behaviour and the CTAs to deposit displayed therein
- [ ] The list of assets is sorted in descending order of balance held (in quantum terms) 

E2e:
- [ ] I can see the markets each is currently traded in, up to a max of 5 sorted by top volume 

## Tasks

- [ ] Detailed sketch
- [ ] Visual design
- [ ] Frontend tasks:
    - [ ] Implement UI
    - [ ] Integrate with backend
    - [ ] Write unit tests
- [ ] Backend tasks:
    - [ ] TBC
    - [ ] Write unit tests
- [ ] Write end to end tests
- [ ] Code review
- [ ] Team verify acceptance criteria met
- [ ] Exploratory QA testing 
- [ ] Capture bugs and new edge cases in tickets (this one or new TBD) 
- [ ] Write automated regression tests for the A/Cs (expanding on A/Cs where possible) 
- [ ] Add story and A/Cs to spec doc 
- [ ] Review with stakeholders
- [ ] Release

### LINKED ISSUE #580: Assets view updates based on feedback
## Chore

Updates based on feedback:

- [ ] Market lozenges, should be code, not name
- [ ] Move markets traded in into the expanded view

### PR DESCRIPTION
**Pull request by dexturr**
Closes #562 
Closes #580
Closes https://github.com/vegaprotocol/vegawallet-browser/issues/555
Closes https://github.com/vegaprotocol/vegawallet-browser/issues/425

And address error cases for when there is an error fetching data from a data node.

### CODE REVIEW

File: frontend/stores/markets-store.ts [hunk 1]
```diff
@@ -3,17 +3,7 @@ import { RpcMethods } from '../lib/client-rpc-methods.ts'
...
-export const getSettlementAssetId = (market: vegaMarket) => {
-  const assetId =
-    get(market, 'tradableInstrument.instrument.future.settlementAsset') ??
-    get(market, 'tradableInstrument.instrument.perpetual.settlementAsset')
-  if (!assetId) {
-    throw new Error(`Could not find settlement asset from market ${market.id}`)
-  }
-  return assetId
-}
+import { getSettlementAssetId } from '../lib/markets.ts'
```
Comment (dexturr) [inline frontend/stores/markets-store.ts:6]:
```suggestion
import { getSettlementAssetId } from '../lib/markets'
```

**Comment by dalebennett1992** [comment]
so to my eyes, I do not see how this AC is addressed
As a user
I want to see a reasonable empty state
So that I am informed about how to deposit assets to vega and which markets I can trade in

Here is my view with no balance
image

here is the view when expanded
image

if we are aiming to meet the above AC I am not sure any of the changes do this. Of course I could be missing something

**Comment by dalebennett1992** [comment]
Regarding the error handling and getting this view
image

The above works if these details have been retrieved once. So if I load successfully, then kill the mock server and open key details for a second time I will see the above. However, this still bombs out if the assets endpoint does not respond in the first instance.
image

I managed to get that by commenting out my /assets response so the info could not be retrieved on the initial load. Should we address this?

**Comment by dalebennett1992** [comment]
When I send a transaction with a node completely down (no mock running) I get this error
image

Here is the error message
`"Error: Asset with id fc7fd956078fb1fc9db5c19b88f0874c4299b2a7639ad05a47a28c0aef291b55 not found\n    at getAssetById (chrome-extension://jfaancmgehieoohdnmcdfdlkblfcehph/popup.js:112044:20)\n    at useFormatAssetAmount (chrome-extension://jfaancmgehieoohdnmcdfdlkblfcehph/popup.js:112441:24)\n    at EnrichedTransferView (chrome-extension://jfaancmgehieoohdnmcdfdlkblfcehph/popup.js:112455:42)\n    at renderWithHooks (chrome-extension://jfaancmgehieoohdnmcdfdlkblfcehph/popup.js:21100:20)\n    at mountIndeterminateComponent (chrome-extension://jfaancmgehieoohdnmcdfdlkblfcehph/popup.js:24864:15)\n    at beginWork (chrome-extension://jfaancmgehieoohdnmcdfdlkblfcehph/popup.js:26377:18)\n    at beginWork$1 (chrome-extension://jfaancmgehieoohdnmcdfdlkblfcehph/popup.js:32201:16)\n    at performUnitOfWork (chrome-extension://jfaancmgehieoohdnmcdfdlkblfcehph/popup.js:31335:14)\n    at workLoopSync (chrome-extension://jfaancmgehieoohdnmcdfdlkblfcehph/popup.js:31241:7)\n    at renderRootSync (chrome-extension://jfaancmgehieoohdnmcdfdlkblfcehph/popup.js:31209:9)"`

asset id should be irrelevant given the node is completely dead, but for what it's worth here is the transfer obj
```
export const dummyTransaction = {
  fromAccountType: 4,
  toAccountType: 4,
  amount: '1',
  asset: 'fc7fd956078fb1fc9db5c19b88f0874c4299b2a7639ad05a47a28c0aef291b55',
  to: 'fc7fd956078fb1fc9db5c19b88f0874c4299b2a7639ad05a47a28c0aef291b55',
  kind: {
    oneOff: {}
  }
}
```


Here is the test code (it fails at the point of the static wait), I think this is a bug.
`it('shows an error when no nodes are available', async () => {
    closeServer = false
    await navigateToExtensionLandingPage(driver)
    await viewWallet.openKeyDetails('Key 1')
    const errorText = await keyDetails.getNotificationText()
    expect(errorText).toBe('An error occurred when loading account information: Failed to fetch data')
    await vegaAPI.connectWalletAndCheckSuccess()
    await connectWallet.checkOnConnectWallet()
    await connectWallet.approveConnectionAndCheckSuccess()
    const keys = await vegaAPI.listKeys()
    await vegaAPI.sendTransaction(keys[0].publicKey, { transfer: dummyTransaction })
    await staticWait(30000)
    await transaction.checkOnTransactionPage()
  })`

**Review by dalebennett1992 — CHANGES_REQUESTED** [review]
all green with passing resilience tests however we will want to get to the bottom of why the warning seems to now ALWAYS show... if I copy the error I do not get anything either (this is in a scenario where everything should be ok)

**Comment by dalebennett1992** [comment]
all tests that should pass now pass, the failing tests is due to a legit bug. If we get that final test passing then this can go down. Over to you @dexturr ;)

**Review by dalebennett1992 — APPROVED** [review]
